app/models/work.rb

```diff
@@ -12,6 +12,7 @@ class InvalidCollectionError < ::ArgumentError; end
   has_many_attached :pre_curation_uploads, service: :amazon_pre_curation
 
   belongs_to :collection
+  belongs_to :curator, class_name: "User", foreign_key: "curator_user_id", optional: true
```

Comment:
Is there any argument for accepting the default foreign key? We've just consistently said `<role>_user_id` instead of `<role>_id`, so we should stick with it?

Reply:
This was keeping the code the same and just changing the mechanism.  We may want to ticket changing the name if we think it is desired.  That will impact many more places.